lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7266,7 +7266,7 @@ void ClangImporter::diagnoseMemberValue(const DeclName &name,
                                                     nominalTypesToLookInto);
   for (auto containerDecl : nominalTypesToLookInto) {
     const clang::Decl *clangContainerDecl = containerDecl->getClangDecl();
-    if (clangContainerDecl && isa<clang::DeclContext>(clangContainerDecl)) {
+    if (isa_and_nonnull<clang::DeclContext>(clangContainerDecl)) {
```

Comment:
Accidental change?

Reply:
`isa_and_nonnull` is more idiomatic in LLVM for checks like this, but I am happy to remove this if we don't want to land small cleanups along with features.

Reply:
Yeah, I try to commit these changes separately, especially if they aren't in the vicinity of the code that's being changed. That makes it slightly easier to review and cherry-pick things. I don't think it's a super strict rule, but the Swift contributing guide says this:

> Large or invasive changes usually have secondary changes that must be made before the large change (for example, API cleanup or addition). Commit these changes before the major change, independently of that work.

https://www.swift.org/contributing/#contributing-code

Reply:
I'll open a separate PR with this change.